edx_arch_experiments/kafka_consumer/docs/decisions/0002_kafka_consumer_command.rst

```diff
@@ -1,15 +1,14 @@
 Managing Kafka Consumers
---------------
+========================
 
 Status
-======
-
-In Progress
+------
+Provisional
 
 Context
-=======
+-------
 As outlined in the upcoming OEP-52, edX.org has elected to go with Apache Kafka as our event bus implementation. Though the decision presented here is predicated on this particular edX.org decision, it is included to help other Open edX users evaluate Kafka for their own purposes. The standard pattern for consuming events with Kafka is to poll in a loop and process messages as they come in. According to the Confluent team it is a best practice to limit each consumer to a single topic (Confluent is a platform for industry-scale Kafka management)::
```

Comment:
Are we going to keep the Kafka decision in OEP-52 or will that just talk about the general architecture after your revisions?

Reply:
Good catch. Although the OEP will reference the ADR, I updated this text to reference the ADR explicitly instead.